test/WebSites/ValidationWebSite/Controllers/ModelMetadataTypeAttributeController.cs

```diff
@@ -38,6 +38,14 @@ public object ValidateSoftwareViewModelIncludingMetadata([FromBody] SoftwareView
             return CreateValidationDictionary();
         }
 
+        [HttpPost]
+        public object TryValidateModelWithCollectionsModel([FromBody] List<ProductViewModel> products)
```

Comment:
yes, because of the prefix that is used in case of from body and from form can be different.